src/log_config.py

```diff
@@ -35,7 +35,7 @@
 # Formatter
 formatter = logging.Formatter("%(asctime)s %(name)-12s %(levelname)-8s %(message)s")
 
-for handler in (ch, fh):
+for handler in (ch, fh):  # type: ignore
```

Comment:
Why do we have to ignore this error?

Reply:
I never worked with `mypy` before, but this is the error without ignoring the type:
```
src\log_config.py:38:1: error: Incompatible types in assignment (expression has type "object", variable has type "Handler")  [assignment]  
Found 1 error in 1 file (checked 13 source files)
```
I tried to find a solution online, but no luck. I think it has something to do with handler being defined earlier already.

Reply:
Ok, I do not understand why this error was never recognized before... But it looks like the type von log.handlers is Iterable[object]. Reassigning it to a handler variable in the second loop raises the mypy error. 

We could assert is instance(handler, logging.handler.Handler) in the first loop which should solve this.

Reply:
damn.. i'll give it a try when reviewing.